src/com/facebook/buck/android/toolchain/ndk/impl/NdkCxxPlatforms.java

```diff
@@ -268,6 +264,7 @@ public static boolean isSupportedConfiguration(Path ndkRoot, NdkCxxRuntime cxxRu
 
     // ARM Platform
```

Comment:
Github is not letting me comment on the actual code line in this file, but can we change
```
  public static final ImmutableSet<String> DEFAULT_CPU_ABIS =
      ImmutableSet.of("arm", "armv7", "x86");
```

to 

```
  public static final ImmutableSet<String> DEFAULT_CPU_ABIS =
      ImmutableSet.of("armv7", "x86");
```

That will make buck work by default with the latest ndk

Reply:
https://github.com/facebook/buck/pull/1894